### DISCUSSION TITLE
Display "x days ago" indicator for pending major dependency upgrades

### DISCUSSION THREAD

**Discussion by mcmxcdev** [renovatebot/renovate#22597] [Suggest an Idea]
### Tell us more.

We have set up `stabilityDays: 30` for any major version upgrade. Some dependencies definitely need to have that timeframe to become stable and allow dependent library authors to update to it, but other dependencies could easily be merged after a shorter time frame.

It would be great to have a small indicator next to each entry showing when the release of that major version happened. That way, I can ignore the ones that were released e.g. < 7d ago and be quite confident to merge the ones that are more matured already.

![image](https://github.com/renovatebot/renovate/assets/16797721/c4d53e41-e683-4922-9dbb-efa31220c5b7)

**Comment by jamietanna** [comment]
This sounds like a good option (for what's now called `minimumReleaseAge`)

One concern I see with relative dates are that it depends on when Renovate last ran + updated the Dependency Dashboard, not based on something dynamically updated by i.e. GitHub/GitLab/etc when a user loads the page 🤔 

So there's the slight risk someone may read it, feel it's "stable enough", and it's actually a few days later than that - in that case, it's probably a positive, as it'll then be more stable, so perhaps not a concern
